README.md

```diff
@@ -215,8 +198,14 @@ the FV3 makefiles:
 
 And build the model
 
-    cd FV3
-    make
+    make -C FV3
+
+Then, to run some simple tests execute
```

Comment:
is emulation going to be a long term thing in this repo? if so it might be useful to describe it in the readme as well. But of course that's something that can be added later as it solidifies.

Reply:
There are some docs, but they are kind of hidden here: https://github.com/ai2cm/fv3gfs-fortran/tree/master/tests/emulation.

Would be easier to follow if we had some sphinx docs for this repo (see #224).